sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ParallelDocumentQueryTest.java

```diff
@@ -539,4 +547,44 @@ private void queryWithContinuationTokensAndPageSizes(String query, int[] pageSiz
 
         return receivedDocuments;
     }
+
+    //TODO: Fix the test for GW mode
```

Comment:
Do we still need to fix this for gateway ?

Reply:
later this needs to be fixed. for some reason doesn't work now.